docs/v1/open_questions.md

```diff
@@ -0,0 +1,112 @@
+# Open Questions
+
+This document contains open questions regarding the
+[API design document](./design.md).
+
+## 1. Is the `ServiceInstanceBinding` resource necesssary?
+
+The `ServiceInstanceBinding` object is a byproduct of the successful execution
+of a `ServiceInstanceClaim`. Instead of writing a new `ServiceInstanceBinding`
+object, can the service catalog controller just add the binding information
+directly into the `ServiceInstanceClaim` on success?
+
+## Should we write Kubernetes references for binding information?
+
+Currently, the information in a `ServiceInstanceBinding` that points to created
+resources looks like this:
+
+```yaml
+secrets:
+  - my-zip-code-creds-password
+configMaps:
+  - my-zip-code-creds-conn-info
+```
+
+Instead, should we write pointers to these resources using Kubernetes reference
+notation, as below?
+
+```yaml
+resources:
+  - /api/v1/namespaces/${NAMESPACE}/secrets/my-zip-code-creds-password
+  - /api/v1/namespaces/${NAMESPACE}/configmaps/my-zip-code-creds-conn-info
+```
+
+## 2. Should `ServiceInstance`s be namespaced?
+
+If `ServiceInstance`s were namespaced, then operators could enforce their
+security with [RBAC](http://kubernetes.io/docs/admin/authorization/#rbac-mode).
+
+If we do namespace `ServiceInstance`s, however, a few questions arise:
+
+1. It would become messy for pods to share a `ServiceInstance` inter-namespace
+1. A consumer may still submit a `ServiceInstanceClaim` that refers to a
+  `ServiceInstance` that is in a RBAC-forbidden namespace.
+  1. Possible solution: the controller enforces security, as is the case with
```

Comment:
Imagine that the controller lowers its privileges to those of the user before accessing the ServiceInstance on the user's behalf. The access should fail in this case. An error earlier, such as when creating the ServiceInstanceClaim (in admission control), would be more user-friendly.